### ISSUE TITLE
IServiceCollection Extension AddOData namespace

### ISSUE THREAD

**Issue by grahamehorner** [OData/WebApi#1216]
The IServiceCollection extension method AddOData namespace is currently Microsoft.**AspNet**.OData.Extensions IMHO this should be Microsoft.Extensions.DependencyInjection and follow the standard that other Microsoft extension methods using eg. EntityFrameWork, Logging, Configuration, Caching, ...

**Comment by robertmclaws** [comment]
Yep. Just to point out to the OData devs, the convention for extension methods is to put the class in the namespace you're extending. If you have to import a different namespace for an extension methods, it's not very discoverable.
